pkg/analyzer/analyzer.go

```diff
@@ -51,6 +52,8 @@ var (
 	blueprintpRegexTargetScope                      = regexp.MustCompile("(\\s*\"targetScope\":)|(\\s*targetScope:)")
 	blueprintpRegexProperties                       = regexp.MustCompile("(\\s*\"properties\":)|(\\s*properties:)")
 	buildahRegex                                    = regexp.MustCompile(`\s*buildah\s*from\s*\w+`)
+	dockerComposeVersionRegex                       = regexp.MustCompile(`\s*version:`)
+	dockerComposeServicesRegex                      = regexp.MustCompile(`\s*services:`)
```

Comment:
```suggestion
	dockerComposeVersionRegex                       = regexp.MustCompile(`\s*version\s*:`)
	dockerComposeServicesRegex                      = regexp.MustCompile(`\s*services\s*:`)
```
[This PR](https://github.com/Checkmarx/kics/pull/4857/files) improved the other regexes